.github/workflows/release.yml

```
@@ -112,6 +114,70 @@ jobs:
+          VERSION="${{ github.ref_name }}"
+          VERSION=${VERSION#v}  # Remove 'v' prefix if present
+          echo "Updating package.json to version: $VERSION"
+          npm version "$VERSION" --no-git-tag-version --allow-same-version
+
+      - name: 🚀 Deploy to Production
+        env:
+          RENDER_DEPLOY_HOOK: ${{ secrets.RENDER_DEPLOY_HOOK }}
+          NORTHFLANK_DEPLOY_HOOK: ${{ secrets.NORTHFLANK_DEPLOY_HOOK }}
+        run: |
+          echo "🚀 Deploying to production environments..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)